BlinkyHeadless/Python/PythonBlinkyHeadless/PythonBlinkyHeadless.pyproj

```diff
@@ -8,7 +8,7 @@
     <SchemaVersion>2.0</SchemaVersion>
     <ApplicationType>Windows Store</ApplicationType>
     <WindowsTargetPlatformVersion>10.0.10586.0</WindowsTargetPlatformVersion>
-    <WindowsTargetPlatformMinVersion>10.0.10240.0</WindowsTargetPlatformMinVersion>
+    <WindowsTargetPlatformMinVersion>10.0.10586.0</WindowsTargetPlatformMinVersion>
```

Comment:
Yes, no changes in the source change.  The magic is in the .dll (.pyd); it will detect and auto switch between inbox drivers and DMAP drivers.